# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1215541328 14400
#      Tue Jul 08 14:22:08 2008 -0400
# Node ID c51ae36fcbce829ad019e13b6803dc95eb4240ff
# Parent  3e4c9b69069df9f6522415b9872ba649de6491f9
graphics.h.in (axes::properties::keypos): Declare as any_property instead of double_property

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2008-07-08  John W. Eaton  <jwe@octave.org>
+
+	* graphics.h.in (axes::properties::keypos): Declare as
+	any_property instead of double_property.
+
 2008-06-28  John W. Eaton  <jwe@octave.org>
 
 	* debug.cc (push_dummy_call_stack_elt): Delete.
 	(Fdbstack): Don't push dummy stack elt.  Use nskip instead of
 	nframes, and curr_frame instead of idx.  Pass curr_frame to
 	octave_call_stack::backtrace.
 
 	* ls-mat5.cc (read_mat5_binary_element): Explicitly pass 0 for
diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -2593,17 +2593,17 @@ public:
     // properties declarations.
 
     BEGIN_PROPERTIES(axes)
       array_property position u , default_axes_position ()
       mutable handle_property title GSO , graphics_handle ()
       bool_property box , "on"
       bool_property key , "off"
       bool_property keybox , "off"
-      double_property keypos , 1
+      any_property keypos , 1
       array_property colororder , default_colororder ()
       array_property dataaspectratio m , Matrix (1, 3, 1.0)
       radio_property dataaspectratiomode , "{auto}|manual"
       radio_property layer , "{bottom}|top"
       row_vector_property xlim mu , default_lim ()
       row_vector_property ylim mu , default_lim ()
       row_vector_property zlim mu , default_lim ()
       row_vector_property clim m , default_lim ()
